test/unit/math/prim/meta/require_generics_test.cpp

```diff
@@ -221,58 +221,6 @@ TEST(requires_prim_scal, var_or_arithmetic_test) {
                        int>::any_not();
 }
 
-/**
- * Require container
- */
-TEST(requires_prim_arr, container_type_test) {
-  EXPECT_FALSE((stan::container_value_type_check_base<
-                stan::is_vector, std::is_floating_point, std::string>::value));
-  EXPECT_TRUE(
-      (stan::container_value_type_check_base<stan::is_vector,
-                                             std::is_floating_point,
-                                             std::vector<double>>::value));
-}
-
-template <template <class> class ContainerCheck,
-          template <class> class TypeCheck, class Check, typename = void>
-struct require_container_vt_tester : std::false_type {};
-
-template <template <class> class ContainerCheck,
-          template <class> class TypeCheck, class Check>
-struct require_container_vt_tester<
-    ContainerCheck, TypeCheck, Check,
-    stan::require_container_vt<ContainerCheck, TypeCheck, Check>>
-    : std::true_type {};
-
-TEST(requires_prim_arr, generic_container_vt_test) {
-  EXPECT_FALSE(
-      (require_container_vt_tester<stan::is_vector, std::is_floating_point,
-                                   double>::value));
-  EXPECT_TRUE(
-      (require_container_vt_tester<stan::is_vector, std::is_floating_point,
-                                   std::vector<double>>::value));
-}
-
-template <template <class> class ContainerCheck,
-          template <class> class TypeCheck, class Check, typename = void>
-struct require_container_st_tester : std::false_type {};
-
-template <template <class> class ContainerCheck,
-          template <class> class TypeCheck, class Check>
-struct require_container_st_tester<
-    ContainerCheck, TypeCheck, Check,
-    stan::require_container_st<ContainerCheck, TypeCheck, Check>>
-    : std::true_type {};
-
-TEST(requires_prim_arr, generic_container_st_test) {
-  EXPECT_FALSE(
-      (require_container_st_tester<stan::is_vector, std::is_integral,
-                                   std::vector<std::vector<double>>>::value));
-  EXPECT_TRUE(
-      (require_container_st_tester<stan::is_vector, std::is_floating_point,
-                                   std::vector<std::vector<double>>>::value));
-}
-
```

Comment:
Why are those tests deleted?

Reply:
These were checking the `stan::require_container_vt` and `stan::require_container_st` which do not exist anymore